scripts/fetch_ext_bins.sh

```diff
@@ -31,7 +31,7 @@ k8s_version=1.11.0
 goarch=amd64
 goos="unknown"
 
-if [[ "$OSTYPE" == "linux-gnu" ]]; then
+if [[ "$OSTYPE" == "linux"* ]]; then
```

Comment:
what the other linux we are trying to support beside `linux-gnu`?

Reply:
in my local machine, OSTYPE is just `linux`.